docker/dependencies/compute_dependencies_hash.sh

```diff
@@ -9,4 +9,4 @@
 # Exit the script at the first failure
 set -e
 
-find dependencies docker/dependencies -not -wholename '*/trilinos_develop/*' -not -name 'README.md' -type f -exec sha1sum {} \; | sort | sha1sum | cut -c -8
+find dependencies docker/dependencies -not -wholename '*/trilinos_develop/*' -not -wholename '*/dealii/*' -not -name 'README.md' -type f -exec sha1sum {} \; | sort | sha1sum | cut -c -8
```

Comment:
It may make sense to explicitly name what to folders to check instead of excluding all other folders.
```suggestion
find dependencies/current dependencies/testing docker/dependencies -not -name 'README.md' -type f -exec sha1sum {} \; | sort | sha1sum | cut -c -8
```
This should give you the same files. Please double-check